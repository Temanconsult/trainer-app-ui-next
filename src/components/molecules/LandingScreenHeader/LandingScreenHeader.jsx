import React from 'react'
import { Navbar, Nav, Button } from 'react-bootstrap'
import routeTo from '../../../utils/routeTo'

const LandingScreenHeader = () => {
  return (
    <Navbar bg="light" expand="lg" className="mt-sm-4">
      <Navbar.Brand onClick={() => routeTo('/')}>Best Tutors</Navbar.Brand>
      <Navbar.Toggle aria-controls="basic-navbar-nav" />
      <Navbar.Collapse id="basic-navbar-nav">
        <Nav className="mr-auto">
          <Nav.Link onClick={() => routeTo('/')}>Home</Nav.Link>
          <Nav.Link onClick={() => routeTo('/about')}>About</Nav.Link>
          <Nav.Link onClick={() => routeTo('/courses')}>Courses</Nav.Link>
          <Nav.Link onClick={() => routeTo('/tutors')}>Tutors</Nav.Link>
        </Nav>
        <Nav>
          <Nav.Link onClick={() => routeTo('/signup')}>
            Become and Instructor
          </Nav.Link>
          <Button
            variant="outline-primary"
            className="mr-lg-3 mb-lg-0 mb-sm-3"
            onClick={() => routeTo('/login')}
          >
            Login
          </Button>
          <Button variant="primary" onClick={() => routeTo('/signup')}>
            SignUp
          </Button>
        </Nav>
      </Navbar.Collapse>
    </Navbar>
  )
}

export default LandingScreenHeader
