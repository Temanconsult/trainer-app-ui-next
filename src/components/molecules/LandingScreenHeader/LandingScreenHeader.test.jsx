import React from 'react'
import { render } from '@testing-library/react'
import LandingScreenHeader from '.'

describe('LandingScreenHeader', () => {
  it('creates landingg screen top bar', () => {
    const { asFragment } = render(<LandingScreenHeader />)
    expect(asFragment()).toMatchSnapshot()
  })
})
