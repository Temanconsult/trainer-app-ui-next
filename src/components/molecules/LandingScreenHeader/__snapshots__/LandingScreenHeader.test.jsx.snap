// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingScreenHeader creates landingg screen top bar 1`] = `
<DocumentFragment>
  <nav
    class="mt-sm-4 navbar navbar-expand-lg navbar-light bg-light"
  >
    <span
      class="navbar-brand"
    >
      Best Tutors
    </span>
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="basic-navbar-nav"
    >
      <div
        class="mr-auto navbar-nav"
      >
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Home
        </a>
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          About
        </a>
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Courses
        </a>
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Tutors
        </a>
      </div>
      <div
        class="navbar-nav"
      >
        <a
          class="nav-link"
          href="#"
          role="button"
        >
          Become and Instructor
        </a>
        <button
          class="mr-lg-3 mb-lg-0 mb-sm-3 btn btn-outline-primary"
          type="button"
        >
          Login
        </button>
        <button
          class="btn btn-primary"
          type="button"
        >
          SignUp
        </button>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
